client/packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -77,6 +77,7 @@
   "messages.cant-delete-requisitions": "Can only delete requisitions with a status of 'Draft'",
   "messages.changing-max-mos": "This will change the maximum months of stock target.",
   "messages.changing-min-mos": "This will change the minimum months of stock threshold.",
+  "messages.unassign-min-mos": "This will remove the minimum months of stock threshold. Minimum MOS will default to be the same as maximum MOS.",
```

Comment:
Can you rephrase all this since we're renaming min and max MOS in UI, I think it makes more sense to change all these messages to target + reorder threshold... and having both min/max and threshold/target makes it sound a bit repetitive...

'This will remove the stock threshold. Target months of stock will now be used as the threshold.' or something similar...